lib/git-lfs-s3/services/ceph_presigner.rb

```diff
@@ -0,0 +1,24 @@
+require 'base64'
+require 'date'
+require 'digest/sha1'
+require 'openssl'
+require 'uri'
+
+module GitLfsS3
+  module CephPresignerService
+    extend self
+    extend AwsHelpers
+
+    def signed_url(obj)
```

Comment:
Also as an aside. There are some differences between normal request header authorization and presigned URLs. Ceph S3 (radosgw) does v2 request header authorization correctly. It just doesn't supprot presigned URLs. Which is a small subset of the overall functionality, but important to git-lfs.